### PR TITLE
Backbone log event ids

### PR DESCRIPTION
**Pull request by NikolaVetnic**
Added a table documenting all of the event ids assigned to specific log statements, which were introduced in order to facilitate search and distinction of significant events.

Updated the `_data/navigation.yml` file to contain the newly added table.

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -145,6 +145,10 @@ docs_operate:
         url: /operate/privacy-considerations
       - title: "Performance Considerations"
         url: /operate/performance-considerations
+      - title: Backbone Operations
+        children:
+          - title: "Log Event IDs"
+            url: /operate/backbone-log-event-ids
```
Comment (jkoenig134) [inline _data/navigation.yml:151]: I think the indentation is wrong here.
```suggestion
  - title: Backbone Operations
    children:
      - title: "Log Event IDs"
        url: /operate/backbone-log-event-ids
```

**Review by jkoenig134 — DISMISSED** [review]

File: _includes/scenarios/operate/backbone-log-event-ids.md
Comment (jkoenig134) [inline _includes/scenarios/operate/backbone-log-event-ids.md:1]: this file should be in `_includes/scenarios/operate/log-event-ids.md`

Comment (jkoenig134) [reply _includes/scenarios/operate/backbone-log-event-ids.md:1]: @NikolaVetnic this is also crashing the build

File: _data/navigation.yml [hunk 2]
```diff
@@ -145,6 +145,10 @@ docs_operate:
         url: /operate/privacy-considerations
       - title: "Performance Considerations"
         url: /operate/performance-considerations
+  - title: Backbone Operations
+    children:
+      - title: "Log Event IDs"
+        url: /operate/backbone-log-event-ids
```
Comment (jkoenig134) [inline _data/navigation.yml:151]:
```suggestion
        url: /operate/log-event-ids
```

File: _includes/scenarios/operate/backbone-log-event-ids.md
Comment (NikolaVetnic) [reply _includes/scenarios/operate/backbone-log-event-ids.md:1]: So simply a file rename?

Comment (jkoenig134) [reply _includes/scenarios/operate/backbone-log-event-ids.md:1]: yeah, I think so

**Review by jkoenig134 — APPROVED** [review]

**Review by vinzlange — DISMISSED** [review]

**Comment by tnotheis** [comment]
FYI: the codes in this docs will soon update due to a code review process currently going on in the source code repository.

**Review by jkoenig134 — APPROVED** [review]

**Review by vinzlange — APPROVED** [review]

**Review by vinzlange — APPROVED** [review]

**Review by tnotheis — APPROVED** [review]

**Pull request merged**